python/interpret_community/dataset/dataset_wrapper.py

```diff
@@ -19,6 +21,50 @@
     from shap.common import DenseData
 
 
+class CustomTimestampFeaturizer(BaseEstimator, TransformerMixin):
+    def __init__(self, features):
+        self._features = features
+        return
+
+    def fit(self, X):
+        # If the data was previously successfully summarized, then there are no
+        # timestamp columns as it must be numeric.
+        # Also, if the dataset is sparse, we can assume there are no timestamps
+        if isinstance(X, DenseData) or sp.sparse.issparse(X):
+            return self
+        tmp_dataset = X
+        # Temporarily convert to pandas for easier and uniform timestamp handling
+        if isinstance(X, np.ndarray):
+            tmp_dataset = pd.DataFrame(X, columns=self._features)
+        self._time_col_names = [column for column in tmp_dataset.columns if is_datetime(tmp_dataset[column])]
+        # Calculate the min date for each column
+        self._min = []
+        if self._time_col_names:
+            for time_col_name in self._time_col_names:
+                self._min.append(tmp_dataset[time_col_name].map(lambda x: x.timestamp()).min())
+        return self
+
+    def transform(self, X):
+        tmp_dataset = X
+        # Temporarily convert to pandas for easier and uniform timestamp handling
+        if isinstance(X, np.ndarray):
+            tmp_dataset = pd.DataFrame(X, columns=self._features)
+        if self._time_col_names:
```

Comment:
Explicitly change to make it clear that this is a list

Reply:
done!